Tao_Current.txt

```diff
@@ -163,133 +187,159 @@ The IETF discussion list is unmoderated. This means that all can express their o
+
 3. IETF Meetings
 
-The computer industry is rife with conferences, seminars, expositions, and all manner of other kinds of meetings. IETF face-to-face meetings are nothing like these. The meetings, held three times a year, are week-long "gatherings of the tribes" whose primary goal is to reinvigorate the WGs to get their tasks done, and whose secondary goal is to promote a fair amount of mixing between the WGs and the Areas. The cost of the meetings is paid by the people attending and by the corporate host for each meeting (if any), although IASA kicks in additional funds for things such as the audio broadcast of some Working Group sessions.
+The computer industry is rife with conferences, seminars, expositions, and all manner of other kinds of meetings. IETF face-to-face meetings are not like these. The meetings, held three times a year, are week-long gatherings whose primary goal is to reinvigorate the WGs to get their tasks done, and whose secondary goal is to promote a fair amount of mixing between the WGs and the Areas. The cost of the meetings is paid by the people attending and by the corporate host for each meeting (if any), although IASA kicks in additional funds for things such as the remote participation.
 
 For many people, IETF meetings are a breath of fresh air when compared to the standard computer industry conferences. There is no exposition hall, few tutorials, and no big-name industry pundits. Instead, there is lots of work, as well as a fair amount of time for socializing for many participants. IETF meetings are of little interest to sales and marketing folks, but of high interest to engineers and developers.
 
-The general flow of an IETF meeting is that it begins with tutorials and an informal gathering on Sunday, and that there are WG and BoF meetings Monday through Friday. WG meetings last for between 1 and 2.5 hours each, and some WGs have meetings multiple times during the week, depending on how much work they anticipate doing.
+The general flow of an IETF meeting is that it begins with a hackathon on Saturday and Sunday, tutorials and an informal gathering on Sunday, and that there are WG and BoF meetings Monday through Friday. WG meetings last for between 1 and 2.5 hours each, and some WGs have meetings multiple times during the week, depending on how much work they anticipate doing.
+
+There are two plenary sessions, one technical and one administrative, in the evenings during the week (mostly Wednesday). The technical plenary is organized by the IAB and usually has one or two panels of experts on topics of interest across many WGs and Areas. The administrative plenary, organized by the IETF Chair, covers things like progress reports from the RFC Editor and announcements of upcoming meetings. The plenaries are a good time to share with the IESG and IAOC. Praise is welcome, but more often concerns and gripes are raised. In recent meetings, the two plenaries have been combined.
```

Comment:
Note: we often merge the plenaries.

Reply:
Is merging better than 'combining' ? Suggest we keep existing text.